addons/mail/static/src/messaging/component/chat_window_hidden_menu/chat_window_hidden_menu.js

```diff
@@ -111,7 +122,10 @@ class ChatWindowHiddenMenu extends Component {
      * @param {mail.messaging.entity.ChatWindow} ev.detail.chatWindow
      */
     _onClickedChatWindow(ev) {
-        ev.detail.chatWindow.makeVisible();
+        const chatWindow = ev.detail.chatWindow;
+        chatWindow.makeVisible();
+        chatWindow.unfold();
+        chatWindow.focus();
         this.env.messaging.chatWindowManager.closeHiddenMenu();
```

Comment:
Would be better as just dispatch in entity than successive synchronous entity statements inside component.